src/pages/AboutPage.tsx

```
@@ -0,0 +1,19 @@
+
+export default function AboutPage() {
+  const html = useMarkdown("/about.md");
+
+  return (
+    <div className="max-w-3xl mx-auto px-4 py-8">
+      <Card>
+        <CardHeader>
+          <CardTitle>About +chorus</CardTitle>
+        </CardHeader>
+        <CardContent className="prose dark:prose-invert">
+          <div dangerouslySetInnerHTML={{ __html: html }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)